src/compiler/crystal/tools/formatter.cr

```diff
@@ -2060,14 +2060,15 @@ module Crystal
       end
 
       # This is the case of an enum member
-      if node.name[0].ascii_uppercase? && @token.type == :","
-        write ","
+      # TODO: remove comma support after 0.28.0
```

Comment:
Support for commas should stay regardless.

Reply:
This is not the consensus from https://github.com/crystal-lang/crystal/issues/7608, though no dead line is decided. It could be pushed further.

Reply:
Having this TODO is certainly better than not, even if we later decided to keep this conversion indefinitely. But I don't think that's reasonable. We don't need to throw it out in the next version, but eventually it would just be useless dead code.